### PR TITLE
Issues/226 fix cwl options

### PR DESCRIPTION
**Pull request by wshands**
This fixes mistakes in the CWL option strings and also adds the reference files to the parameterized JSON

### CODE REVIEW

**Comment by arkal** [comment]
I thought we agreed you're going to add signed links from redwood for each input reference file. Hardcoding the values in the wrapper is a very bad way to do this.

File: docker/protect.cwl [hunk 1]
```diff
@@ -52,7 +52,7 @@ dct:creator:
 
 requirements:
   - class: DockerRequirement
-    dockerPull: "quay.io/ucsc_cgl/protect:2.5.1-1.12.3"
+    dockerPull: "quay.io/ucsc_cgl/protect:2.5.2-1.12.3"
```
Comment (arkal) [inline docker/protect.cwl]: latest 2.5.3 docker and cwl reflect the correct value

File: docker/protect.cwl [hunk 2]
```diff
@@ -63,49 +63,49 @@ hints:
 
 inputs:
 
-  sample-name:
+  sample_name:
```
Comment (arkal) [inline docker/protect.cwl]: Why?

File: docker/protect.json [hunk 3]
```diff
@@ -1,36 +1,147 @@
 {
```
Comment (arkal) [inline docker/protect.json]: I thought we agreed you're going to add signed links from redwood for each input reference file. Hardcoding the values in the wrapper is a very bad way to do this.

Comment (wshands) [inline docker/protect.json]: The protect.json is just an example of how to construct a parameterized json; I used an example if running with a dockstore command where the files are on the local machine. In the core platform the parameterized JSON is constructed with a S3 path that consists of a bundle uuid, and file uuid and file name

File: docker/protect.cwl (hunk 2 above)
Comment (wshands) [inline docker/protect.cwl]: I switched to using underscores to separate the cwl identifiers from the ProTECT switch names which us hyphens. There were a couple of cwl identifiers that had hyphens and I changed them to underscores to make everything uniform

File: docker/protect.json (hunk 3 above)
Comment (arkal) [inline docker/protect.json]: That was not evident in this. Add a multi-line comment at teh top stating this, and add it to the README

File: docker/protect.cwl (hunk 2 above)
Comment (arkal) [inline docker/protect.cwl]: So this is syntactic

File: MANUAL.md [hunk 4]
```diff
@@ -488,3 +488,12 @@ results. Every docker image required for the run is described in required_docker
 required tool is also hosted freely on the dockerhub `aarjunrao` (Hence the default value in the
 config). If you wish to use a personal repo instead, ensure that every required version of every
 tool is made available on the repo.
+
+ProTECT is registered on Dockstore.org and can be run using Dockstore with the following command:
+```
+dockstore tool launch --debug --entry quay.io/ucsc_cgl/protect:<version tag>  --json protect.json
+```
+The example protect.json file provided in this repo uses local file paths on the assumption that
+the input files are on the local file system, and includes a minimal set of reference files for
```
Comment (arkal) [inline MANUAL.md]: The example protect.json file provided in this repo describes a run where the input reference files are on the local file system at `/home/ubuntu/protect-work/protect-reference-files/hg38-reference-files` and the input fastqs are in `/home/ubuntu/protect-work/samples/`.

**Pull request merged**